spring/pim-tool-back/src/main/java/vn/elca/training/model/entity/Group.java

```diff
@@ -0,0 +1,27 @@
+package vn.elca.training.model.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "tGROUP")
```

Comment:
[N] You can escape table name
https://thorben-janssen.com/hibernate-tips-escape-table-column-names/